stdlib/math.jou

```diff
@@ -0,0 +1,41 @@
+declare abs(x: int) -> int
+declare llabs(x: long) -> long
+declare fabs(x: double) -> double
+
+declare ceil(x: double) -> double
+declare floor(x: double) -> double
+declare round(x: double) -> double
+
+declare acos(x: double) -> double
+declare asin(x: double) -> double
+declare atan(x: double) -> double
+declare atan2(x: double, y: double) -> double
+declare cos(x: double) -> double
+declare sin(x: double) -> double
+declare tan(x: double) -> double
+
+declare fmod(x: double, y: double) -> double
+declare sqrt(x: double) -> double
+declare cbrt(x: double) -> double
+
+declare log(x: double) -> double
+declare log10(x: double) -> double
+declare log2(x: double) -> double
+
+declare exp(x: double) -> double
+declare exp2(x: double) -> double
+declare pow(x: double, y: double) -> double
+declare pow10(p: int) -> double
```

Comment:
We should probably use `exp10` instead of `pow10`. The manual page of `pow10` says:

```
       Note  well:  These functions perform exactly the same task as the func‐
       tions described in exp10(3), with the difference that the latter  func‐
       tions  are now standardized in TS 18661-4:2015.  Those latter functions
       should be used in preference to the functions described in this page.
```

Also, please add a test for all functions. There is no test for `pow10`.